File: Week_5/problem_2-Print.py
# https://codeforces.com/group/MWSDmqGsZm/contest/223205/problem/B

# Function to print numbers from 1 to n
def print_fun(n):
    for i in range(1, n + 1):
        print(i, end=" " if i < n else "")

n = int(input())    # Reading input
print_fun(n)        # Calling the function

File: Week_5/problem_2-Print.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/223205/problem/B

#include <bits/stdc++.h>
using namespace std;

void print_fun(int n)
{
    for (int i = 1; i <= n; i++)
    {
        printf("%d", i);
        if (i < n)
            printf(" ");
    }
}

int main()
{
    int n;
    cin >> n;

    print_fun(n);

    return 0;
}
